packages/dbml-core/src/parse/mssql/constraint_definition/index.js

```diff
@@ -60,11 +60,14 @@ const Lang = P.createLanguage({
     A.makeDefaultConstraint,
   ),
 
-  ConstExpr: () => P.seq(
-    KP.LParen.fallback(null),
+  ConstExpr: (r) => P.alt(
+    P.seq(
+      KP.LParen,
+      r.ConstExpr,
+      KP.RParen,
+    ).map(value => value[1]),
     P.alt(pFunction, pConst, KP.KeywordNull.thru(streamline('boolean'))),
```

Comment:
You only need to use one `P.alt()` here. Suggest modify ConstExpr to below and the PR should be good to to.
```
ConstExpr: (r) => P.alt(
    pFunction,
    pConst,
    KP.KeywordNull.thru(streamline('boolean')),
    P.seq(
      KP.LParen,
      r.ConstExpr,
      KP.RParen,
    ).map(value => value[1]),
  ),
```